tools/dependency/BUILD

```diff
@@ -0,0 +1,26 @@
+load("@rules_python//python:defs.bzl", "py_binary")
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
```

Comment:
not sure if this should be an `envoy_package()`

Reply:
This is fine, it's just setting visibility (you can skip it on the targets).

Reply:
targets updated